pkg/storage/unified/resource/index_mapping.go

```diff
@@ -25,6 +26,7 @@ type IndexedResource struct {
 }
 
 func (ir IndexedResource) FromSearchHit(hit *search.DocumentMatch) IndexedResource {
+	ir.Uid = hit.Fields["Uid"].(string)
 	ir.Kind = hit.Fields["Kind"].(string)
 	ir.Name = hit.Fields["Name"].(string)
 	ir.Namespace = hit.Fields["Namespace"].(string)
```

Comment:
might panic if the field doesn't exist or is nil.  not sure if that could ever happen

Reply:
yeah, I think that's true for all these. But, they all _should_ exist afaik. Good enough for the PoC, but we should add some more graceful error handling for unpacking the search results at some point.